app/components/primer/popover_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render Primer::BaseComponent.new(**@kwargs) do %>
+  <%= render Primer::BaseComponent.new(**@message_kwargs) do %>
+    <%= render Primer::BaseComponent.new(**@heading_kwargs) do %>
+      <%= heading %>
+    <% end %>
+    <%= body %>
+    <%= render Primer::BaseComponent.new(**@button_kwargs) do %>
```

Comment:
Maybe I don't even want this button, but should leave it to the caller to include in `body` if they want. 🤔